cmd/apps/istio_app.go

```diff
@@ -114,11 +114,21 @@ func MakeInstallIstio() *cobra.Command {
 		outputPath := path.Join(chartPath, "istio")
 
 		if initIstio, _ := command.Flags().GetBool("init"); initIstio {
-			err = helm3Upgrade(outputPath, "istio/istio-init", namespace, "", defaultVersion, overrides, wait)
+			// Waiting for the crds to appear
+			err = helm3Upgrade(outputPath, "istio/istio-init", namespace, "", defaultVersion, overrides, true)
 			if err != nil {
 				return fmt.Errorf("unable to istio-init install chart with helm %s", err)
 			}
 
+			err = kubectl("-n", "istio-system", "wait", "--for=condition=complete", "job", "--all")
```

Comment:
Why is this required? Isn't it part of the `--wait` flag that is optional on apps like `openfaas` and `inlets-operator`?

Reply:
This are two different points.

I will remove the `kubectl` command, it is not needed, but we have to force wait here, if not the second install step will fail. Is it okay for you?